src/constants/properties.constant.ts

```diff
@@ -0,0 +1,23 @@
+const DEFAULTS = {
+  PARTY: {
+    partyTypeCode: '100',
+    address: {
+      addressIdentifier: 'PRM',
+      addressName1: 'PRV',
+      addressTypeCode: '01',
+    },
+    generalLedgerUnitIdentifier: 'ECGD',
+    riskRatingCode: '14',
+    lineOfficerIdentifier: 'DCIS',
+    servicingUnitSectionIdentifier: 'ACBS',
+    servicingUnitIdentifier: 'ACBS',
+    partyUserDefinedList1Code: '',
+    partyUserDefinedList2Code: '',
+    partyUserDefinedList3Code: '',
+    languageCode: 'ENG',
+    partyStatusCode: 'AC',
+    watchListReasonCode: '',
+  },
+};
```

Comment:
We've structured this a bit differently in our other open PRs, instead we've gone for:

```
const PROPERTIES = {
  PARTY: {
    DEFAULT: {
      <values here>
    }
  }
}
```